src/main/webapp/style.css

```diff
@@ -1,9 +1,169 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+/*--------------
+Imported Fonts
+---------------*/
+@import url('https://fonts.googleapis.com/css?family=Montserrat:100,200,300,400,500,600,700,800,900');
+@import url('https://fonts.googleapis.com/css?family=Sriracha');
```

Comment:
In order to get the checks to pass I think you can instead import these via `<link>` tags in the HTML:
https://developers.google.com/fonts/docs/getting_started#a_quick_example